pyproject.toml

```diff
@@ -46,18 +46,26 @@ select = [
     "E", # pycodestyle Error
     "F", # Pyflakes
     "W", # pycodestyle Warning
-    # PYI: only enable rules that always autofix, avoids duplicate # noqa with flake8-pyi
+    # PYI: only enable rules that have autofixes and that we always want to fix (even manually),
+    # avoids duplicate # noqa with flake8-pyi and flake8-noqa flagging `PYI` codes
     # See https://github.com/plinss/flake8-noqa/issues/22
```

Comment:
If anyone has a contact to @plinss , that's be a nice flake8-noqa feature to get merged ^^"